examples/graph-demo/index.html

```diff
@@ -343,9 +524,9 @@
       {collapse: false,
        disable: false,
        media: 'https://unpkg.com/blockly/media/',
-       toolbox: document.getElementById('toolbox')});
-  Blockly.Xml.domToWorkspace(document.getElementById('startBlocks'),
-                             Graph.workspace);
+       toolbox: toolbox});
+       
```

Comment:
These extra spaces may be a lint error when CI finishes running:)

Reply:
well, maybe not?

Reply:
we don't actually lint examples 😅

Reply:
I removed them:D